contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -2493,6 +2513,157 @@ is_char_identpart(char c)
 			(c >= '0' && c <= '9'));
 }
 
+/*
+ * Put delimiters around a T-SQL variable/parameter that is
+ * named '@@var' or contains a hash, e.g. '@var#'.
+ * Without delimiters, the backend will raise an error.
+ * This is used for the parameter argument of sp_executesql, so the input
+ * string may contain multiple names, e.g.: @par1 int, @par2 varchar(20), ...
+ */
```

Comment:
I think that this header should be updated to clarify that this function returns a newly palloc'ed string.

Reply:
added comment